src/components/SearchBar/SearchBar.jsx

```diff
@@ -1,20 +1,35 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
+import { usePromiseTracker } from 'react-promise-tracker';
+import { useHistory } from 'react-router-dom';
+
+import { geolocated } from 'react-geolocated';
+
+import { withStyles } from '@material-ui/core/styles';
+
 import {
   Box,
   IconButton,
-  InputBase,
+  TextField,
   Paper,
-  Typography,
 } from '@material-ui/core';
+
 import SearchIcon from '@material-ui/icons/Search';
-import { useHistory } from 'react-router-dom';
-import { withStyles } from '@material-ui/core/styles';
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+import Autocomplete, { createFilterOptions } from '@material-ui/lab/Autocomplete';
+
+import States from '../../api/states';
+import useSearch from '../../routes/Search/hooks/useSearch';
```

Comment:
We can move this to the global hooks folder so that we can reuse it across components (`/src/hooks` )